stdlib/public/core/Indices.swift

```diff
@@ -82,6 +82,16 @@ extension DefaultIndices: Collection {
   public var indices: Indices {
     return self
   }
+  
+  public func contains(_ element: Element) -> Bool {
```

Comment:
Can you add this as `_customContainsEquatableElement(_:) -> Bool?` instead of `contains(_:) -> Bool`? That way the optimization will still get used in generic contexts.

Reply:
Sure.